api/controlplane/kubeadm/v1beta2/kubeadm_control_plane_types.go

```diff
@@ -444,29 +444,13 @@ type KubeadmControlPlaneSpec struct {
 	// +optional
 	KubeadmConfigSpec bootstrapv1.KubeadmConfigSpec `json:"kubeadmConfigSpec,omitempty,omitzero"`
 
-	// rolloutBefore is a field to indicate a rollout should be performed
-	// if the specified criteria is met.
-	// +optional
-	RolloutBefore *RolloutBefore `json:"rolloutBefore,omitempty"`
-
-	// rolloutAfter is a field to indicate a rollout should be performed
-	// after the specified time even if no changes have been made to the
-	// KubeadmControlPlane.
-	// Example: In the YAML the time can be specified in the RFC3339 format.
-	// To specify the rolloutAfter target as March 9, 2023, at 9 am UTC
-	// use "2023-03-09T09:00:00Z".
+	// rollout defines the rollout behavior.
```

Comment:
Nit: I would normally expect something more detailed than this

Reply:
This one in particular seems tautological. I would normally expand on what the field allows you to do as a user, so in this case talk about configuring options like the strategy that are within rollout, like a brief mention rather than anything else

So maybe something like:

```
rollout allows you to configure the behaviour of rolling updates to the control plane Machines.
It allows you to require that all machines are replaced before or after a certain time, and allows you to define
the strategy used during rolling replacements.
```

The point here is that if I `kubectl explain` the spec, I want to see what I can do in each section, rather than just something basic